onboarding/views.py

```diff
@@ -678,12 +702,12 @@ def finished(self, request, pk):
         """
         if request.method == 'PATCH':
             answers = Answer.objects.filter(section__page__id=pk,
-                owner=self.request.user) # id__in=request.data['answers']
+                                            owner=self.request.user)  # id__in=request.data['answers']
```

Comment:
one extra tab is enough I think, but we go in right direction 😊

Reply:
IDEs complaining :wink: